src/backend/InvenTree/part/test_api.py

```diff
@@ -2668,7 +2682,13 @@ def test_output_options(self):
         """Test that various output options work as expected."""
         self.run_output_test(
             reverse('api-bom-item-detail', kwargs={'pk': 3}),
-            ['can_build', 'part_detail', 'sub_part_detail'],
+            [
+                'can_build',
+                'part_detail',
+                'sub_part_detail',
+                # TODO re-enable 'substitutes',
+                # TODO re-enable ('pricing', 'pricing_min'),
```

Comment:
These are disabled as they are not working in `master` right now too - this would be a follow-up PR